### PR TITLE
chore: update metamask version

### PR DESCRIPTION
**Pull request by Lykhoyda**
Updated recommended metamask version

**Short description of work done**

### PR Checklist

- [X] I have run linter locally
- [X] I have run unit and integration tests locally
  - [ ] Update configuration the newest version (readme and const)
- [ ] Rebased to master branch / merged master

### Changes

- _EXAMPLE - Removed basic auth from the controller_

### Issues

Closes #

### CODE REVIEW

**Comment by Tbaut** [comment]
The flask doesn't seem happy with the "Connect" button. Did they change the name?

**Comment by mpetrunic** [comment]
> The flask doesn't seem happy with the "Connect" button. Did they change the name?

Could be

**Comment by Lykhoyda** [comment]
> The flask doesn't seem happy with the "Connect" button. Did they change the name?

No, they didn't change, it's failing only on CI

**Review by BeroBurny — APPROVED** [review]
LGTM
note: 20h ago released `10.23.0` but will be put in a week, maybe we could use it now.

File: test/dapp/data.js [hunk 1]
```diff
@@ -0,0 +1,65 @@
+const ContractInfo = {
```
Comment (Tbaut) [inline test/dapp/data.js:1]: This is not needed any more.

**Comment by Lykhoyda** [comment]
> o we have retry?
> This error is usually related to popups which are a tricky thing as we need to refresh after they are ready to ensure we bring them to the front so checkout retry policy if we have one there.

Tried to add retry, but it doesn't work on CI. We probably should create an story for that

**Review by BeroBurny — APPROVED** [review]
LGTM

**Review by Tbaut — APPROVED** [review]

**Pull request merged**